casper/src/main/scala/coop/rchain/casper/ValidatorIdentity.scala

```diff
@@ -59,11 +56,6 @@ object ValidatorIdentity {
     )
   }
 
-  def getEnvVariablePassword[F[_]: Sync: EnvVars]: F[String] =
-    EnvVars[F].get(RNodeValidatorPasswordEnvVar) >>= (
-      _.liftTo(new Exception(s"Environment variable $RNodeValidatorPasswordEnvVar is unspecified"))
-    )
```

Comment:
This is not used now, but later we can provide more comprehensive way to manage private keys for validators.